source/server-selection/server-selection-tests.rst

```diff
@@ -296,20 +298,28 @@ Multi-threaded and async drivers MUST also implement the following prose test:
         },
     }
 
-3. Create a client with both mongoses' addresses in its seed list,
-   appName="loadBalancingTest", and command monitoring enabled.
+3. Insert an empty document into the collection being used for the test.
 
-4. Start 10 concurrent threads / tasks that each run 10 `findOne` operations
-   using that client.
+4. Create a client with both mongoses' addresses in its seed list,
+   appName="loadBalancingTest", and localThresholdMS=30000.
 
-5. Assert that fewer than 25% of the CommandStartedEvents occurred on the mongos
-   that the failpoint was enabled on.
+5. Using CMAP events, ensure the client's connection pools for both
```

Comment:
This helps reduce any noise introduced by connection establishment latency during the actual server selection tests.